jetty-server/src/main/config/etc/jetty-ssl.xml

```diff
@@ -44,6 +44,7 @@
     <Call name="addCustomizer">
       <Arg>
         <New class="org.eclipse.jetty.server.SecureRequestCustomizer">
```

Comment:
Should both SecureRequestCustomizer and SslContextFactory implement a common interface to add the setSNIRequired/isSNIRequired methods?

Reply:
hhmmmmm maybe.  But I don't see the value in that for now.